### PR TITLE
Fix test snap icons, readmes and snap manifests to be uniform

### PR DESCRIPTION
**Pull request by bowensanders**
See #175

### CODE REVIEW

File: packages/error/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Error Test Snap
+
+A MetaMask Test Snap that throws an error.
```
Comment (FrederikBolding) [inline packages/error/README.md:3]:
```suggestion
A MetaMask Test Snap that throws an out-of-band error.
```

File: packages/insights/images/icon.svg
Comment (FrederikBolding) [inline packages/insights/images/icon.svg:1]: Should we use the template icon for all the test-snaps instead? https://github.com/MetaMask/template-snap-monorepo/blob/main/packages/snap/images/icon.svg

Comment (bowensanders) [reply packages/insights/images/icon.svg:1]: Updated the icons :)

**Review by FrederikBolding — APPROVED** [review]

**Review by FrederikBolding — CHANGES_REQUESTED** [review]
Looks like the manifests need to be regenerated after updating the icons.

**Review by FrederikBolding — APPROVED** [review]

**Pull request merged**